src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -89,6 +105,12 @@ module sonic-flex_counter {
                 leaf FLEX_COUNTER_DELAY_STATUS {
                     type flex_delay_status;
                 }
+                leaf POLL_INTERVAL {
+                    type uint32 {
+                        range 100..30000;
+                    }
+                    default 1000;
```

Comment:
By setting default this goes into CONFIG_DB and we don't want it which makes it tough to differentiate from user configuration. There was a consensus in YANG subgroup not to have default in yang. Please remove the default.

Reply:
@dgsudharsan Do you mean to remove the default for the PORT container or for all containers?

Reply:
The default value for interval is not required. It is handled by application in the backend. Please remove the 'default' that is introduced in this PR

Reply:
The 'default' values have been removed.